content/docs/5.0/get-started/tikv-in-5-minutes.md

```diff
@@ -0,0 +1,209 @@
+---
+title: TiKV in 5 Minutes
+description: Get started using TiKV in 5 Minutes
+menu:
+    "5.0":
+        parent: Get Started
+        weight: 1
+---
+
+## Prerequisites
+
+Ensure that you are using MacOS or Linux.
+
+{{< warning >}}
+The following steps are only for test environments.
+{{< /warning >}}
+<!-- TODO: For production environments please refer to ??? -->
+
+## Step1: Install a Playground TiKV Cluster
+
+1. Install TiUP by executing the following command:
+```bash
+$ curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+```
+
+2. Set the TiUP environment variables:
+
+Redeclare the global environment variables:
+
+```bash
+$ source .bash_profile
+```
+
+Confirm whether TiUP is installed:
+```bash
+$ which tiup
+```
+
+3. If TiUP is already installed, update the TiUP playground component to the latest version:
+```bash
+$ tiup update --self && tiup update playground
+```
+
+4. Use TiUP playground to start a local TiKV cluster
+```bash
+$ tiup playground --mode tikv-slim
+```
+
+5. Press `Ctrl +C` to stop the local TiKV cluster
+
+{{< info >}}
+Refer to [TiUP playground document](https://docs.pingcap.com/tidb/stable/tiup-playground) to find more TiUP playground commands.
+{{< /info >}}
+
+## Step2: Monitor the cluster
+
+Open [http://127.0.0.1:3000](http://127.0.0.1:3000) and login to the Grafana (username: admin password: admin).
+
+
+Open `TiKV-Summary` page on Grafana to find the metrics of your TiKV cluster.
+
+## Step3: Hello, World!
+
+### Python
+
+1. Install `tikv-client` python package
+
+```bash
+$ pip3 install -i https://test.pypi.org/simple/ tikv-client
+```
+
+{{< info >}}
+This package requires Python 3.5+.
+{{< /info >}}
+
+2. Try `TXN KV` API
+
+Write test code
+
+```bash
+$ echo '
+from tikv_client import TransactionClient
+
+client = TransactionClient.connect("127.0.0.1:2379")
+
+txn = client.begin()
+txn.put(b"k1", b"Hello")
+txn.put(b"k2", b",")
+txn.put(b"k3", b"World")
+txn.put(b"k4", b"!")
+txn.put(b"k5", b"TXN KV")
+txn.commit()
+
+snapshot = client.snapshot(client.current_timestamp())
+print(snapshot.get(b"k1"))
+print(snapshot.batch_get([b"k1", b"k3"]))
+print(snapshot.scan(b"k1", end=b"k5", limit=10, include_start=True, include_end=True))
+' > test_txn.py
+```
+
+Run test script
+
+```bash
+$ python3 test_txn.py
+
+b'Hello'
+[(b'k3', b'World'), (b'k1', b'Hello')]
+[(b'k1', b'Hello'), (b'k2', b','), (b'k3', b'World'), (b'k4', b'!'), (b'k5', b'TXN KV')]
+```
+
+3. Try `RAW KV` API
```

Comment:
I prefer to introduce the Raw API firstly

Reply:
agree